toonz/sources/toonzqt/tonecurvefield.cpp

```diff
@@ -460,6 +460,64 @@ void ChennelCurveEditor::removeCurrentControlPoint() {
 
 //-----------------------------------------------------------------------------
 
+void ChennelCurveEditor::selectNextControlPoint()
```

Comment:
I believe 'Chennel' should likely be 'Channel' here and in other places in this code.
The spelling doesn't appear to effect any processing but I'll look into that too.